src/main/kotlin/no/nav/syfo/services/UpdateInfotrygdService.kt

```diff
@@ -100,7 +103,9 @@ class UpdateInfotrygdService {
                 RULE_HIT_STATUS_COUNTER.labels(validationResultBehandler.status.name).inc()
                 log.warn("Behandler er ikke register i HPR")
                 produceManualTaskAndSendValidationResults(kafkaproducerCreateTask, receivedSykmelding, validationResultBehandler,
-                        navKontorManuellOppgave, loggingMeta, oppgaveTopic, sm2013BehandlingsUtfallToipic, kafkaproducervalidationResult)
+                        navKontorManuellOppgave, loggingMeta, oppgaveTopic, sm2013BehandlingsUtfallToipic, kafkaproducervalidationResult,
+                        InfotrygdForespAndHealthInformation(infotrygdForespResponse, healthInformation),
+                        "LE", jedis)
```

Comment:
Hva er "LE"? Kan kanskje lage en konstant av den med et navn som sier hva det er? :)

Reply:
Retter det til å benytte enum, det står for Lege